foundation-extensions/experience-platform-extension/README.md

```diff
@@ -10,27 +10,36 @@ The Adobe Experience Platform Edge Network extension requires the Identity for E
 
-![Edge Network extension configuration](../../.gitbook/assets/mobile-edge-launch-configuration.png)
+![Edge Network extension configuration](../../.gitbook/assets/mobile-edge-ui-configuration.png)
 
 ### Datastreams
 
-Select the `Datastream` from the drop-down list. Once you do that, the Production, Staging and Development environments will be automatically prefilled. If no datastream was previously created, see [Configure datastreams](../../getting-started/configure-datastreams.md).
+If no datastream was previously created, see [Configure datastreams](../../getting-started/configure-datastreams.md) before moving to the next step.
 
-If you use multiple Development configurations, select the desired one from the `Development environment` drop-down.
+You may configure only the required datastream for the production environment, and by default the staging and development environments will use the same datastream. Alternatively, if you want to use different datastreams per each environment, select the desired ones from the corresponding drop-down lists.
+
+{% hint style="info" %}
+If your organization uses multiple sandboxes, select the **Sandbox** first, then select the **Datastream** for each environment.
+{% endhint %}
 
 The datastream used by the client-side implementation is one of the followings:
 
 * the `Production environment` configuration when the Launch library is published to production \(in the Published column in the Launch publishing flow\).
 * the `Staging environment` configuration when the Launch library is published to staging \(in the Submitted column in the Launch publishing flow\).
 * the `Development environment` configuration when the Launch library is in development.
 
-### Edge Network domain
+### Domain configuration
+
+The value under the **Edge Network domain** field is used for requests to Adobe Experience Platform Edge Network and it usually follows the format `<company>.edge.adobedc.net`, where `<company>` is the unique namespace associated to your Adobe organization.
+
+If you have a first-party domain mapped to the Adobe-provisioned Edge Network domain, you can enter it here. For more details about how to configure or maintain a first-party domain, see [Adobe-Managed Certificate Program](https://experienceleague.adobe.com/docs/core-services/interface/administration/ec-cookies/cookies-first-party.html?lang=en#adobe-managed-certificate-program).
 
-If you have a first-party domain mapped to the Adobe-provisioned Edge Network domain, enter it here. The domain name is expected to be just the domain without any protocol or trailing slash. To use the default Adobe Edge Network domain, leave this field blank.
+**Note:** The domain name is expected to be just the domain without any protocol or trailing slashes. If no domain is provided, by default the `edge.adobedc.net` domain is used.
```

Comment:
I thought it will default to < company >.data.adobedc.net , not < company >.edge.adobedc.net?